src/crimson/osd/osd_operation.h

```diff
@@ -52,14 +52,15 @@ class OperationT : public Operation {
 
   virtual ~OperationT() = default;
 
+  uint64_t maybe_set_prev_op(uint64_t last_issued) {
```

Comment:
if an op experiences a pg interval change before it completes, we should not update its prev op id. as the last issued id is not its prev op id anymore. so it can (and should!) be a part of the construction if allowed.

yes, the sequencer is a per osd-connection facility, i will add a commit to set the prev op id when constructing an ClientRequest.